rasa/utils/io.py

```diff
@@ -11,6 +12,8 @@
 import ruamel.yaml as yaml
 from io import BytesIO as IOReader, StringIO
 
+import simplejson
```

Comment:
Why are we using simplejson? For the more frequent updates?

Reply:
has better error reporting on failed load / dump